src/pages/Earn/index.tsx

```diff
@@ -59,20 +62,20 @@ export function Earn(isPfx: boolean) {
           <CardSection>
             <AutoColumn gap="md">
               <RowBetween>
-                <TYPE.white fontWeight={600}>{isPfx ? 'Polarfox' : 'gAKITA'} liquidity mining</TYPE.white>
+                <TYPE.white fontWeight={600}>{rewardToken.name} liquidity mining</TYPE.white>
               </RowBetween>
               <RowBetween>
                 <TYPE.white fontSize={14}>
-                  Deposit your Polarfox Liquidity Provider PFX-LP tokens to receive{' '}
-                  {isPfx ? 'PFX, the Polarfox' : 'gAKITA, the Akita Inu'} governance token.
+                  Deposit your Polarfox Liquidity Provider PFX-LP tokens to receive {rewardToken.symbol}, the{' '}
+                  {rewardToken.name} governance token.
```

Comment:
Won't work with gAKITA as its name is "Akita Inu Governance Token". We also want the "governance token" part to be displayed in lower case.